_docs/tutorial/00-setup.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Setup
+overview: The required setup instructions for the tutorial.
+
+order: 00
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+1. Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [curl](https://curl.haxx.se/download.html), [node.js](https://nodejs.org/en/download/), [Docker](https://docs.docker.com/install/)
+and get access to a [Kubernetes](https://kubernetes.io) cluster.
+For example, you can try the [IBM Cloud Container Service](https://console.bluemix.net/docs/containers/container_index.html#container_index).
```

Comment:
should we put all or none of the providers ? or the main contributor's providers (ie both GCP and IBM)?

Reply:
I propose that we start with this PR with IBM. Then we add all other providers as separate PRs, by the provider people. If you think we should add GCP right now, please submit a PR to my branch with GCP instructions.

Reply:
done. thx